### ISSUE TITLE
Artisan command not found

### ISSUE THREAD

**Issue by kennyray** [Vinelab/youtube#13]
In your instructions, you say to run `php artisan config:publish Vinelab/youtube`, but when I do, I get the following error. I'm running Laravel 5.2.

> [Symfony\Component\Console\Exception\CommandNotFoundException]
>     Command "config:publish" is not defined.
>   Did you mean one of these?
>       vendor:publish
>       config:clear
>       config:cache

Am I perhaps doing something wrong, or does this command need to be different for the newer version of Laravel?

**Comment by kennyray** [comment]
I just tried `vendor:publish` and got this:

> php artisan vendor:publish
> Copied File [/vendor/vinelab/youtube/src/config/youtube.php] To [/config/youtube.php]
> Publishing complete for tag []!

Is this the correct way to do it under 5.2 now?

**Comment by Mahmoudz** [comment]
Absolutely, that package was built before Laravel 5 and seems the docs needs some update.

**Comment by kennyray** [comment]
Ok. I'm learning as I go here. :-)
